symbiflow.py

```diff
@@ -154,22 +157,27 @@ def max_freq(self):
 
                     fields = l.split(':')
                     group = fields[0].split()[0]
-                    freqs[group] = 1e9 / float(fields[1].split()[0].strip())
+                    freqs[group] = safe_division_by_zero(
+                        1e9, float(fields[1].split()[0].strip())
+                    )
 
         for clk in freqs:
             criticals = self.get_critical_paths(clk, 'setup')
             clocks[clk] = dict()
             clocks[clk]['actual'] = freqs[clk]
             if criticals is not None:
-                clocks[clk]['requested'] = 1e9 / criticals[clk]['requested']
+                clocks[clk]['requested'] = safe_division_by_zero(
+                    1e9, criticals[clk]['requested']
```

Comment:
Why is requested zero?

Reply:
Didn't analyze this thoroughly yet.

I think the issue here is that VPR reports differently the timing information based on whether the design is single- or multi-clocked. I htink it is a matter of correctly parsing the results.